velox/functions/sparksql/Register.cpp

```diff
@@ -282,6 +282,10 @@ void registerFunctions(const std::string& prefix) {
   registerFunction<AddMonthsFunction, Date, Date, int32_t>(
       {prefix + "add_months"});
 
+  registerFunction<DateFromUnixDateFunction, Date, int8_t>({prefix + "date_from_unix_date"});
+  registerFunction<DateFromUnixDateFunction, Date, int16_t>({prefix + "date_from_unix_date"});
```

Comment:
It looks Spark only supports int32 type as input. Right?

Reply:
Yes, updated. Thanks!